Wireframe/index.html

```diff
@@ -1,13 +1,120 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
+
+<head>
```

Comment:
This file shouldn't be in this PR - always ensure you create a _new_ and _clean_ branch for each week's assignment.

Reply:
thanks for the review.